fs/expose/dokan/__init__.py

```diff
@@ -709,17 +728,18 @@ def GetVolumeInformation(self, vnmBuf, vnmSz, sNum, maxLen, flags, fnmBuf, fnmSz
             maxLen[0] = 255
         if flags:
             flags[0] = 0
```

Comment:
The flags probably should be at least `FILE_CASE_SENSITIVE_SEARCH | FILE_CASE_PRESERVED_NAMES | FILE_UNICODE_ON_DISK`. At least without `FILE_CASE_PRESERVED_NAMES` the file listing in Explorer looks weird.